4001-5000/LIT4926HagMir3.xml

```diff
@@ -4,9 +4,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">How Lālibalā became like a poor person</title>
+                <title xml:lang="gez" xml:id="t1">Miracle of Lālibalā: How Lālibalā became a pauper</title>
```

Comment:
en (also in some other records, I think)

Reply:
> en (also in some other records, I think)

indeed! corrected